src/Surfnet/ServiceProviderDashboard/Application/ViewObject/Service.php

```diff
@@ -113,4 +128,20 @@ public function arePrivacyQuestionsEnabled()
     {
         return $this->privacyQuestionsEnabled;
     }
+
+    /**
+     * @return bool
+     */
+    public function hasTestEntities()
+    {
+        return $this->getEntityList()->hasTestEntities();
+    }
+
+    /**
+     * @return bool
+     */
+    public function isProductionEntitiesEnabled()
```

Comment:
Maybe add some return type definitions for the new boolean methods? /nitpick

Reply:
done